deploy/helm/sumologic/values.yaml

```diff
@@ -275,10 +275,12 @@ sumologic:
         ## This feature autodetects how much CPU is assigned to the nginx instance and sets
         ## the right amount of workers based on that. Disable to use the default of 8 workers.
         workerCountAutotune: true
+        ## Nginx listen port
+        port: 8080
       replicaCount: 3
       image:
         ## TODO: Serve this image from the sumologic public ECR
-        repository: public.ecr.aws/nginx/nginx
+        repository: nginxinc/nginx-unprivileged
```

Comment:
We're switching from Public ECR to Docker Hub, which might cause users to hit the Docker Hub transfer limits.

Reply:
Yeah, we'd need to figure that out and ideally not use Docker Hub here.

Reply:
I pushed the copy of the image to `public.ecr.aws/sumologic/nginx-unprivileged`